plugins/data-management/src/ucsc-trackhub/ucscTrackHub.js

```diff
@@ -310,7 +328,29 @@ function makeTrackConfig(
         baseTrackType,
         categories,
       )
+    case 'hic':
+      return {
+        type: 'HicTrack',
+        name: track.get('shortLabel'),
+        description: track.get('longLabel'),
+        category: categories,
+        adapter: {
+          type: 'HicAdapter',
+          hicLocation: bigDataLocation,
+        },
+      }
+    case 'halSnake':
+      return generateUnsupportedTrackConf(
+        track.get('shortLabel'),
+        baseTrackType,
+        categories,
+      )
+
     default:
-      throw new Error(`Unsupported track type: ${baseTrackType}`)
+      return generateUnsupportedTrackConf(
```

Comment:
I think `generateUnknownTrackConf` (also from `@jbrowse/core/util/tracks`) might be better than `generateUnsupportedTrackConf` here. Basically the same thing, but for when you don't recognize the track type, rather than for when you recognize it but don't support it.

Reply:
yeah that sounds better, i changed it to unknown